sass/_navigation.scss

```diff
@@ -221,3 +192,36 @@ a {
         line-height: normal;
     }
 }
+
+.menu .menu-item-has-children {
+    position: relative;
+    
+    .sub-menu-toggle {
+        cursor: pointer;
+        @include rem(font-size, 30px);
+        height: 59px;
+        position: absolute;
+        right: 0;
+        top: 0;
+        width: 60px;
+
+        &:hover {
+            background-color: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.